packages/drivers/odsp-driver/src/odspUtils.ts

```diff
@@ -434,7 +435,7 @@ export function toInstrumentedOdspTokenFetcher(
 										? rawCanRetry
 										: false /* canRetry */,
 									{ method: name, errorMessage, driverVersion },
-								),
+								) as unknown as LoggingError, // Type 'NetworkErrorBasic<"fetchTokenError">' is not assignable to type 'LoggingError'. @fluidframework/odsp-driver:   Types have separate declarations of a private property 'omitPropsFromLogging'.
```

Comment:
``
Type 'NetworkErrorBasic<"fetchTokenError">' is not assignable to type 'LoggingError'. @fluidframework/odsp-driver: Types have separate declarations of a private property 'omitPropsFromLogging'.
``